comment.html

```diff
@@ -0,0 +1,93 @@
+<html>
+    <head>
+        <title>Comments (Have a suggestion?)</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+        <link href='https://fonts.googleapis.com/css?family=Lato:300,400,700' rel='stylesheet' type='text/css'>
+        <link href='custom.css' rel='stylesheet' type='text/css'>
```

Comment:
So I went back to recheck this page and found a few more issues
this **_custom.css_** file does not exist. It is great to create a custom css file for the page but you should check out what the folder structure will be on the master and link to it based on that.